src/admin/cfp.md

```diff
@@ -0,0 +1,37 @@
+# Call for proposals
+
+Each goal milestone corresponds to six months, designated in the format YYYYHN, e.g., 2024H2 or 2025H1. To launch a new goal season, you should get started a month or two before the new season starts:
```

Comment:
```suggestion
Each goal milestone corresponds to six months, designated in the format YYYYhN, e.g., 2024h2 or 2025h1. To launch a new goal season, you should get started a month or two before the new season starts:
```

Reply:
We are I think a bit inconsistent about this.

Reply:
The directions are 2024h1 but other places we write 2024H1, for example.